src/app/containers/FrontPageMain/__snapshots__/index.test.jsx.snap

```diff
@@ -5,6 +5,16 @@ exports[`FrontPageMain should render a pidgin article correctly 1`] = `
   <main
     role="main"
   >
+    <ForwardRef
+      as="h1"
+    >
+      <span
+        lang="en"
+      >
+        Default Brand Name
+      </span>
+       - 
```

Comment:
Also, we should provide a better service config for this snap, with a small subset of dummy translations data to the snap should the desired values render :)